pages/index.md

```diff
@@ -6,38 +6,97 @@ PyCon UK is the annual gathering of the UK Python community and its friends from
+  [NCP Cardiff](https://www.ncp.co.uk/parking-solutions/cities/cardiff)
+  or make other parking arrangements.
+* [A map] of the conference rooms in the City Hall
+
+### Communications
+
+* The Front Desk will be staffed for Registration and general information
+  throughout the conference. Please ask there if you need any help.
+* During the conference the [PyCon UK Slack channel](https://pyconuk-2017.slack.com) 
+  will be used to share announcements and other information
+* You can also follow the [PyCon UK Twitter](https://twitter.com/pyconuk) 
+  account for conference announcements
+* For general queries please contact the [organising committee](/contact/)
 
-The [first four days of the conference](/schedule/) will include talks and workshops,
-while the fifth will be a day of sprints.
+### Food & Drink
 
-We're delighted that the [Education Track](/education/) returns,
-in partnership with the Raspberry Pi Foundation,
-and we look forward to welcoming teachers and children to the conference.
+* Coffee & pastries will be available before the start of the conference
+  but if you want a proper breakfast you'd do well to explore Cardiff's
+  cafe culture.
+* The conference ticket includes lunch which will be available between 1pm and 2.30pm.
+  Since we're feeding upwards of 700 people you will be offered a sitting between those times 
+  when you register. It will help organise things if you can stick to that sitting
+  but we're not doing retina scans at the start of the dinner queue!
+* Coffee & Welsh Cakes will be available during the mid-morning and mid-afternoon
+  breaks.
+* There are two official conference Dinners, one for the whole conference,
+  the other for Contributors (speakers, volunteers etc.). On other nights, or if 
+  you have not booked for those dinners, people tend to self-organise into
+  groups [perhaps using the Slack channel? TJG].
 
-We're also working with [PyData London](http://london.pydata.org/) to put on a
-[track of talks and workshops aimed at data scientists](/pydata/).
+### Atmosphere
 
-It's very important to us that all members of the community feel welcome and
-comfortable at the conference.  [Read about our approach to diversity,
-accessibilty, and inclusion](/diversity-accessibility-inclusion/).
+We very much want PyCon UK to be a welcoming event where everyone feels
+comfortable. Please take the time to read about our approach to 
+[Diversity, Accessibility and Inclusion](/diversity-accessibility-inclusion/).
 
-We've got many more announcements to make over the coming weeks, so you should sign up to our mailing list.
+We ask all those attending the conference to respect all other attendees
+and to honour our [Code of Conduct](/code-of-conduct/).
 
+### Parallel Tracks
+
+PyCon UK is playing host to a number of parallel tracks. These will be
+taking place on different days during the conference and we hope that
+those attending will feel at home in the main conference.
```

Comment:
Some of these (Picademy and Django Girls) require signing up in advance, and are full.  I'd like to rewrite this as follows:

### Other events at PyCon UK

* *[PyData](/pydata/)* (Thursday to Sunday) Talks and workshops about data science, put on in collaboration with [PyData London](https://london.pydata.org/).  Access is included in the price of the conference ticket.

* *[Django Girls](https://djangogirls.org/pyconuk2017/)* (Thursday) A beginner-friendly Python and Django workshop designed to inspire women to fall in love with programming.  Applications are closed.  If you have any questions, please contact the Django Girls organisers by email: pyconuk@djangogirls.org.

* *[Picademy](https://www.raspberrypi.org/training/picademy/)* (Thursday and Friday) A professional development programme for teachers, organised by the [Raspberry Pi Foundation](https://www.raspberrypi.org/).  Applications are closed.  If you have any questions, please [contact the Raspberry Pi Foundation directly](https://www.raspberrypi.org/contact/).

* *[Talks for teachers](http://2017.pyconuk.org/schedule/#saturday)* (Saturday) A day of talks in Room D about Python and education.  Non-teachers are very welcome!

* *[Code Club](/sessions/workshops/code-club/)* (Saturday) Coding workshops for kids aged under 13.  Led by [Code Club](https://www.codeclub.org.uk/) champions and volunteers, we'll be running activities for young coders' using Python to create games and more. There will also be a special session on programming micro:bits and Adafruit's Circuit Playground Express boards - not to be missed!  [Tickets cost £5 and are still on sale](https://hq.pyconuk.org/children/orders/new/)!

* *[Raspberry Jam](/sessions/workshops/raspberry-jam/)* (Saturday) More coding workshops, for young people aged over 13, or anyone looking to do more advanced projects.  The [Raspberry Pi Foundation](https://www.raspberrypi.org/) will be running a Raspberry Jam - and parents and guardians can join in too! We'll be running a variety of digital making activities using Python on the Raspberry Pi.  [Tickets cost £5 and are still on sale](https://hq.pyconuk.org/children/orders/new/)!

* *[Trans*Code](/transcode/)* (Monday) A day-long hackathon to address issues facing the trans* and nonbinary community. The day is open to all trans* and nonbinary people and allies, and we welcome participants from all skill levels and backgrounds.

Reply:
Thanks; I really wasn't sure what to put there. I'll cut-and-paste yours in.